### ISSUE TITLE
In settings documentation for the addon

### ISSUE THREAD

**Issue by Explosion-Scratch** [ScratchAddons/ScratchAddons#475]
Maybe like a details button which would take you to the addon page (stored in the addon) which would have the complete credits, documentation and maybe some screenshots.

**Comment by Explosion-Scratch** [comment]
Like right below the credit: 
![image](https://user-images.githubusercontent.com/61319150/95790539-2aac3c80-0ca5-11eb-9e5e-52ca837d6cdb.png)
Would be this:
![image](https://user-images.githubusercontent.com/61319150/95790560-38fa5880-0ca5-11eb-8210-029bb75debf1.png)
But of course it would look better and not be stolen from google's extension manager.

**Comment by GrahamSH-LLK** [comment]
That'd be great!

**Comment by Explosion-Scratch** [comment]
Thanks! :D

**Comment by Explosion-Scratch** [comment]
I think adding this to 1.2 would be nice!

**Comment by Explosion-Scratch** [comment]
I can help write it if @WorldLanguages approves!

**Comment by WorldLanguages** [comment]
I think @Hans5958 and @JeffaloBob thought about doing this at some point

**Comment by Explosion-Scratch** [comment]
> I think @Hans5958 and @JeffaloBob thought about doing this at some point

Yes, but that was never clear on exactly how this would be done; on YouTube, on the website, etc. I thought that bringing this idea up might help with that.

**Comment by Hans5958** [comment]
I have figured it out the idea, but I still have no time to make the concept on it. Give me a while or two.

**Comment by JeffaloBob** [comment]
yeah i recorded a video for every add-on and stuff but i haven't decided if im ok with putting my voice on the internet with it yet.

and anyways it's outated

**Comment by Hans5958** [comment]
I'd rather have it with texts and no sound.

**Comment by Mbrick2** [comment]
Maybe GIFs
